src/benchmark_metrics.py

```diff
@@ -41,6 +41,9 @@
 MXNET_TRAINING_GPU_IMAGENET_THRESHOLD = 4500
 MXNET_INFERENCE_CPU_IMAGENET_THRESHOLD = 100
 MXNET_INFERENCE_GPU_IMAGENET_THRESHOLD = 4500
+# Accuracy
+MXNET_TRAINING_GPU_IMAGENET_ACCURACY_THRESHOLD = 0.9
+MXNET_TRAINING_GPU_IMAGENET_LATENCY_THRESHOLD = 120
```

Comment:
nit for future PRs: In the case of latency/throughput values, please specify what the units are, such as sec, samples/sec, etc.

Reply:
fixed